module/analytics/src/commonMain/kotlin/org/cru/godtools/shared/analytics/AnalyticsUserProperties.kt

```diff
@@ -0,0 +1,11 @@
+package org.cru.godtools.shared.analytics
+
+object AnalyticsUserProperties {
+    // region Login Status
+    const val LOGGED_IN_STATUS = "cru_loggedinstatus"
```

Comment:
this constant was already defined on Android, I'm moving it here to be consistent